Components/Api.php

```diff
@@ -110,7 +106,8 @@ public function startPayment(\Shopware_Controllers_Frontend_PaynlPayment $contro
         /** @var Payment\Payment $payment */
         $payment = $this->paymentRepository->findOneBy(['name' => $payment_name]);
 
-        $transaction = $this->transactionRepository->createNew($customer, $paymentId, $payment, $signature, $amount, $currency);
+        $transaction =
+            $this->transactionRepository->createNew($customer, $paymentId, $payment, $signature, $amount, $currency);
```

Comment:
I would write
        $transaction = $this->transactionRepository->createNew(
            $customer,
            $paymentId,
            $payment,
            $signature,
            $amount,
            $currency
        );